src/Modules/Meetings/Domain/MeetingCommentingConfigurations/MeetingCommentingConfiguration.cs

```diff
@@ -29,5 +32,27 @@ private MeetingCommentingConfiguration()
 
         internal static MeetingCommentingConfiguration Create(MeetingId meetingId)
             => new MeetingCommentingConfiguration(meetingId);
+
+        public void EnableCommenting(MemberId enablingMemberId, MeetingGroup meetingGroup)
```

Comment:
Please fix the warning *'public' members should come before 'internal' members*

Reply:
sure! i thought i fixed them all.